stream/stream_cdda.c

```diff
@@ -25,13 +25,16 @@
 #define OLD_API
 #endif
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstrict-prototypes"
```

Comment:
Huh what warning is being triggered here? I've seen the others before but I don't recall ever seeing this one.

Reply:
Few functions with empty argument list instead `foo(void)` in `cdio` headers. Probably different version on CI.

Reply:
Which one specifically? I'd expect everyone to be pulling 2.1.0. In fact the `OLD_API` thing is legacy nonsense that should probably be removed.

Reply:
freebsd build see here https://github.com/mpv-player/mpv/actions/runs/6517996634/job/17702986599

Reply:
No idea why that's happening but whatever I guess.